links.html

```diff
@@ -32,7 +32,6 @@
         </ul></nav>
     </header>
     <main>
-        <section class="glossary">
```

Comment:
This does work, but it would be better to just use the #useful-links section you already have here.  See the CSS below.